plugins/docker/client.go

```diff
@@ -1,6 +1,6 @@
 package docker
 
-import "github.com/carlanton/go-dockerclient"
+import "github.com/fsouza/go-dockerclient"
```

Comment:
Same here and in the next file, none of the docker stuff is relevant to this PR.